src/EnsembleKalmanProcess.jl

```diff
@@ -336,4 +368,317 @@ function update_ensemble!(ekp::EnsembleKalmanProcess{FT, IT, Sampler{FT}}, g_in:
+function EnsembleKalmanProcess(
+    obs_mean::Array{FT, 1},
+    obs_noise_cov::Array{FT, 2},
+    process::Unscented{FT, IT};
+    Δt=FT(1)) where {FT<:AbstractFloat, IT<:Int}
+
+#initial parameters stored as columns
+init_params = [DataContainer(update_ensemble_prediction!(process), data_are_columns=true)]
+# Number of parameter
+N_u = length(process.u_mean[1])
+# ensemble size
+N_ens = 2N_u + 1 #stored with data as columns
+#store for model evaluations
+g=[] 
+# error store
+err = FT[]
+# timestep store
+Δt = Array([Δt])
+
+EnsembleKalmanProcess{FT, IT, Unscented}(init_params, obs_mean, obs_noise_cov, N_ens, g, err, Δt, process)
+end
+
+
+
+"""
+EnsembleKalmanProcess Constructor 
+u0_mean::Array{FT} : prior mean
+uu0_cov::Array{FT, 2} : prior covariance
+obs_mean::Array{FT,1} : observation 
+obs_noise_cov::Array{FT, 2} : observation error covariance
+α_reg::FT : regularization parameter toward u0 (0 < α_reg <= 1), default should be 1, without regulariazion
+update_freq::IT : set to 0 when the inverse problem is not identifiable, 
+                  namely the inverse problem has multiple solutions, 
+                  the covariance matrix will represent only the sensitivity of the parameters, 
+                  instead of posterior covariance information;
+                  set to 1 (or anything > 0) when the inverse problem is identifiable, and 
+                  the covariance matrix will converge to a good approximation of the 
+                  posterior covariance with an uninformative prior.
+                  
+"""
+function Unscented(
+    u0_mean::Array{FT, 1}, 
+    uu0_cov::Array{FT, 2},
+    obs_mean::Array{FT,1},
+    obs_noise_cov::Array{FT, 2},
+    α_reg::FT,
+    update_freq::IT;
+    modified_uscented_transform::Bool = true,
+    κ::FT = 0.0,
+    β::FT = 2.0) where {FT<:AbstractFloat, IT<:Int}
+    
+    N_u = size(u0_mean,1)
+    N_y = size(obs_mean, 1)
+    # ensemble size
+    N_ens = 2*N_u+1
+    
+    
+    c_weights = zeros(FT, N_u)
+    mean_weights = zeros(FT, N_ens)
+    cov_weights = zeros(FT, N_ens)
+    
+    # todo parameters λ, α, β
+    
+    α = min(sqrt(4/(N_u + κ)), 1.0)
+    λ = α^2*(N_u + κ) - N_u
+    
+    
+    c_weights[1:N_u]     .=  sqrt(N_u + λ)
+    mean_weights[1] = λ/(N_u + λ)
+    mean_weights[2:N_ens] .= 1/(2(N_u + λ))
+    cov_weights[1] = λ/(N_u + λ) + 1 - α^2 + β
+    cov_weights[2:N_ens] .= 1/(2(N_u + λ))
+    
+    if modified_uscented_transform
+        mean_weights[1] = 1.0
+        mean_weights[2:N_ens] .= 0.0
+    end
+    
+    
+    
+    u_mean = Array{FT,1}[]  # array of Array{FT, 2}'s
+    push!(u_mean, u0_mean) # insert parameters at end of array (in this case just 1st entry)
+    uu_cov = Array{FT,2}[] # array of Array{FT, 2}'s
+    push!(uu_cov, uu0_cov) # insert parameters at end of array (in this case just 1st entry)
+    
+    obs_pred = Array{FT, 1}[]  # array of Array{FT, 2}'s
+    err = FT[]
+    
+    Σ_ω = (2 - α_reg^2)*uu0_cov
+    Σ_ν = 2*obs_noise_cov
```

Comment:
Relating to Melanie's comment of requiring both the observational noise in `EKP` and `Unscented` constructors. It turns out we really don't need these in the `Unscented` object. Note in the constructor it is only used: 
1. `[428] N_y = size(obs_mean, 1)`
2. `[465] Σ_ν = 2*obs_noise_cov`
In the case of 1. we can just provide `N_y` directly. In the case of 2. We could just store the constant "2" and multiply later by the observational noise covariance (stored in `EKP`)- as this matrix is only used in the analysis step of the update.